vpr/src/draw/draw.cpp

```diff
@@ -2593,18 +2505,14 @@ void act_on_mouse_move(ezgl::application* app, GdkEventButton* event, double x,
             std::string msg = vtr::string_fmt("Moused over %s", info.c_str());
             app->update_message(msg.c_str());
         } else {
-            //No rr node moused over, reset message
-            if (!rr_highlight_message.empty()) {
-                app->update_message(rr_highlight_message.c_str());
-            } else {
-                app->update_message(draw_state->default_message);
-            }
+            app->update_message(draw_state->default_message);
```

Comment:
It looks like compared to the previous code this won't actually show the rr_highlight_message?

Reply:
Sorry, I thought I added this variable myself.